csuntechlab/iSTART
# STR-60 -- FE - Create alcohol module thumbnail

# Description
'Write a meaningful description of your task'
  Create the alcohol module thumbnail
# Testing
'What does the reviewer need to do to test your PR?'
 1. Make sure to login as someone with module thumbail. I've been using `april.feldman`
 2. Look at my code make sure it's good, & give help/critique
 3. Make sure it is mobile responsive and pleasing to the eye
 4. Make sure it is close to the mockup. 
# Installation
'Are there any dependencies the reviewer needs to download?'
If you do not see the progress bar loading, then you will need to run `yarn install`
